### ISSUE TITLE
Dataset definition of SemanticPOSS 

### ISSUE THREAD

**Issue by Eaphan** [valeoai/ALSO#10]
Thanks for sharing. Would you please provide the file of SemanticPOSS class? How do you split the train and val set? @aboulch @tristanschultz @CSautier
Thanks in advance.

**Comment by Eaphan** [comment]
"we use the official validation set (sequence 3)."

I got it. You have mentioned in the paper.

**Comment by AlphaSJTU** [comment]
I have the same question: Would you please provide the definition of the SemanticPOSS Dataset class? @aboulch @erjanmx
